trailblazer/server/api.py

```diff
@@ -88,8 +88,8 @@ def refresh_token(auth_service: AuthenticationService = Provide[Container.auth_s
     """Refresh access token."""
     user: User = g.current_user
     try:
-        token: str = auth_service.refresh_access_token(user.id)
-        return jsonify({"access_token": token}), HTTPStatus.OK
+        token: str = auth_service.refresh_token(user.id)
+        return jsonify({"token": token}), HTTPStatus.OK
```

Comment:
One though- you write that we want to return the id token "as well" but this only returns that token right?

Reply:
Ah, sorry that was unclear! I meant "as well" as in we want to return the id token both from `/auth` and `/auth/refresh`